brushfire-core/src/main/scala/com/stripe/brushfire/TraversalStrategy.scala

```diff
@@ -81,6 +93,46 @@ object TraversalStrategy {
   /** Returns an instance of [[MaxWeightedMatch]]. */
   def maxWeightedMatch[K, V, T, A: Ordering]: MaxWeightedMatch[K, V, T, A] =
     new MaxWeightedMatch[K, V, T, A]
+
+  /**
+   * Returns an [[TraversalStrategy]] that always chooses the candidate path
+   * that results in the largest *target* (type `T`). This may have to traverse
+   * each candidate path fully in order to find the resulting target value.
+   *
+   * Optionally, it may be possible to short-circuit some traversals if the
+   * node annotation's contain information about the maximum target in any
+   * descendant leaf nodes. This is specified by providing a `getMaxTarget`
+   * function that returns a value rather than the default of `None`.
+   *
+   * @param getMaxTarget function that returns the maximum target in a subtree
+   */
+  def maxTargetMatch[K, V, T: Ordering, A](getMaxTarget: A => Option[Max[T]] = (_: A) => None): MaxTargetMatch[K, V, T, A] =
+    new MaxTargetMatch[K, V, T, A]({ a =>
+      getMaxTarget(a) match {
+        case Some(max) => Some(max.get)
+        case None => None
+      }
+    })
+
+  /**
+   * Returns an [[TraversalStrategy]] that always chooses the candidate path
+   * that results in the smallest *target* (type `T`). This may have to traverse
+   * each candidate path fully in order to find the resulting target value.
+   *
+   * Optionally, it may be possible to short-circuit some traversals if the
+   * node annotation's contain information about the minimum target in any
+   * descendant leaf nodes. This is specified by providing a `getMinTarget`
+   * function that returns a value rather than the default of `None`.
+   *
+   * @param getMinTarget function that returns the minimum target in a subtree
+   */
+  def minTargetMatch[K, V, T: Ordering, A](getMinTarget: A => Option[Min[T]] = (_: A) => None): MaxTargetMatch[K, V, T, A] =
```

Comment:
can't this just be implemented by using `maxTargetMatch` and reversing the `Ordering`?